tapdb/sqlite.go

```diff
@@ -140,7 +144,41 @@ func NewSqliteStore(cfg *SqliteConfig) (*SqliteStore, error) {
 	// Now that the database is open, populate the database with our set of
 	// schemas based on our embedded in-memory file system.
 	if !cfg.SkipMigrations {
-		if err := s.ExecuteMigrations(TargetLatest); err != nil {
+		backupAndMigrate := func(mig *migrate.Migrate) error {
+			// Create a backup of the database before starting the
+			// migration.
+			backupFullFilePath, err := backupSqliteDatabase(
```

Comment:
Maybe we should add a flag to disable this? Or specify the backup location? Thinking about universes with very large databases where we would now require the available disk space to at least be double.
Though I guess/hope those universes might not be running SQLite...

Reply:
Added flag.